concepts/numbers/introduction.md

```diff
@@ -22,7 +22,7 @@ let plancksConstant : Double = 6.62607015e-34 // plancksConstant is a Double
 ## Arithmetic operators
 
 ~~~~exercism/caution
-In Swift can't mix types in arithmetic operations, so you can't use any arithmetic operator on an `Int` with a `Double` or vice versa.
+In Swift you can't mix types in arithmetic operations, so you can't use any arithmetic operator on an `Int` with a `Double` or vice versa.
```

Comment:
Note, https://github.com/exercism/swift/blob/main/concepts/numbers/about.md?plain=1#L25 has the same typo. I believe these two files are meant to be the same.

Reply:
Yes and https://github.com/exercism/swift/blob/main/exercises/concept/freelancer-rates/.docs/introduction.md